package.json

```diff
@@ -25,7 +25,9 @@
     "prettier": "2.8.1",
     "prettier-eslint": "15.0.1",
     "serve-favicon": "~2.5.0",
-    "uglify-js": "^3.13.4"
+    "uglify-js": "^3.13.4",
```

Comment:
Nit. Could an `npm install` put this in alphabetical order??

Reply:
Doesn't look like it's doing that 😞

Reply:
FYI you can run `npx sort-package-json` to sort package.json. I have this as part of the `@vonage/server-sdk`. It helps curve merge conflicts